lib/src/Uncertainty/StatTests/FittingTest.cxx

```diff
@@ -382,9 +382,9 @@ Scalar FittingTest::BIC(const Sample & sample,
   const Sample logPDF(distribution.computeLogPDF(sample));
   for (UnsignedInteger i = 0; i < size; ++i)
   {
-    if (logPDF(i, 0) == -SpecFunc::MaxScalar) return SpecFunc::MaxScalar;
     logLikelihood += logPDF(i, 0);
   }
+  if (!SpecFunc::IsNormal(logLikelihood)) return SpecFunc::MaxScalar; // catch infinity and NaN
   return (-2.0 * logLikelihood + estimatedParameters * log(1.0 * size)) / size;
```

Comment:
The actual fix should be:
```
const Scalar bic = (-2.0 * logLikelihood + estimatedParameters * log(1.0 * size)) / size;
if (!SpecFunc::IsNormal(bic)) return SpecFunc::MaxScalar;
return bic;
```
otherwise you could get an inf by doubling logLikelihood if its value is large enough.

Reply:
Yes, that is probably what happens under build_linux. Going to fix that.